redfish-core/lib/cable.hpp

```diff
@@ -3,11 +3,64 @@
 #include <query.hpp>
 #include <sdbusplus/asio/property.hpp>
 #include <sdbusplus/unpack_properties.hpp>
+#include <utils/chassis_utils.hpp>
 #include <utils/dbus_utils.hpp>
 #include <utils/json_utils.hpp>
 
 namespace redfish
 {
+
+/**
+ * @brief Api to get Cable Association.
+ * @param[in,out]   asyncResp       Async HTTP response.
+ * @param[in]       cableObjectPath Object path of the Cable with association.
+ * @param[in]       callback        callback method
+ */
+template <typename Callback>
+inline void
+    linkAssociatedCable(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+                        const std::string& cableObjectPath, Callback&& callback)
+{
+    crow::connections::systemBus->async_method_call(
+        [asyncResp, callback{std::forward<Callback>(callback)}](
+            const boost::system::error_code ec,
```

Comment:
include what you use? #include <boost/system/error_code.hpp>

Reply:
Thanks.
It looks like I need to compile.  I'll do it later.
BTW, is this perhaps better to have in pool machine - e.g. gfwa826?